nautobot/core/templates/nautobot_config.py.j2

```diff
@@ -228,13 +228,17 @@ PREFER_IPV4 = False
 RACK_ELEVATION_DEFAULT_UNIT_HEIGHT = 22
 RACK_ELEVATION_DEFAULT_UNIT_WIDTH = 220
 
-# Remote authentication support
-REMOTE_AUTH_ENABLED = False
-REMOTE_AUTH_BACKEND = "nautobot.core.authentication.RemoteUserBackend"
+# Global 3rd-party authentication settings
+EXTERNAL_AUTH_DEFAULT_GROUPS = []
```

Comment:
Looks like in this file we're keeping settings more-or-less in alphabetical order for convenience and consistency. Since these are being renamed, you might want to move them around.

Reply:
Certainly.  Would you like it "alphabetically" after `DATABASES`, `CACHEOPS_REDIS`, `EXEMPT_VIEW_PERMISSIONS`, `DATE_FORMAT`, `DATETIME_FORMAT`, or after `EXTRA_INSTALLED_APPS`?

Reply:
After `EXEMPT_VIEW_PERMISSIONS` looks like the appropriate place for the `EXTERNAL_AUTH_*` settings to me. The `SOCIAL_AUTH_*` settings can go after `SESSION_FILE_PATH`. Thank you!

Reply:
IMO alpha sort in the "NAUTOBOT" section is good enough since these settings are custom. 

For `SOCIAL_AUTH_POSTGRES_JSONFIELD`, it can be in its own section since  that is the only one that is there to explicitly configure the behavior of a the `social_django` plugin?